app/scripts/background.js

```diff
@@ -394,13 +399,18 @@ function overrideContentSecurityPolicyHeader() {
 let connectRemote;
 let connectExternalExtension;
 let connectExternalCaip;
+let connectRemoteCaip;
 
 browser.runtime.onConnect.addListener(async (...args) => {
   // Queue up connection attempts here, waiting until after initialization
   await isInitialized;
 
   // This is set in `setupController`, which is called as part of initialization
   connectRemote(...args);
+
+  if (process.env.MULTICHAIN_API && !isExternallyConnectableWildcardEnabled) {
```

Comment:
I mean honestly, i think `!isManifestV3` is a good enough proxy for determining if this is a chromium build with externally_connectable enabled with wildcards, and that's because all chromium builds are mv3 now, and firefox is mv2. We only have a chromium build and a firefox build, therefore isManifestV3 should be good enough to differentiate between a chromium and firefox build